src/Categories/Category/Cocomplete/Properties/Construction.agda

```diff
@@ -0,0 +1,30 @@
+{-# OPTIONS --without-K --safe #-}
+
+open import Categories.Category
+
+module Categories.Category.Cocomplete.Properties.Construction {o ℓ e} (C : Category o ℓ e) where
+
+open import Level
+
+open import Categories.Category.Construction.Arrow using (Morphism)
+open import Categories.Category.Cocomplete
+open import Categories.Diagram.Coequalizer C
+open import Categories.Diagram.Colimit
+open import Categories.Diagram.Colimit.Properties using (build-colim)
+open import Categories.Functor
+open import Categories.Object.Coproduct.Indexed C
+open import Categories.Object.Coproduct.Indexed.Properties C
+
+private
+  variable
+    o′ ℓ′ e′ : Level
+
+module _ (coprods : AllCoproductsOf (o′ ⊔ ℓ′)) (coequalizer : ∀ {A B} (f g : C [ A , B ]) → Coequalizer f g) where
+  private module _ {J : Category o′ ℓ′ e′} (F : Functor J C) where
```

Comment:
what's the point of putting things in a `private` module here?

Reply:
This has remained from the previous version of the corresponding file for `Complete`. It prevents `cocomplete` from being public. I'm going to inline it.